admin_manual/configuration/server/logging_configuration.rst

```diff
@@ -56,27 +56,54 @@ In addition to the ``%replacements%`` below ``%level%`` can be used, but it is u
 as a dedicated parameter to the syslog logging facility anyway.
 
 ::
+
     'log.syslog.format' => '[%reqId%][%remoteAddr%][%user%][%app%][%method%][%url%] %message%',
-    // 'log.syslog.format' => '{%app%} %message%', // For the old syslog message format
+
+       For the old syslog message format use:
+     
+    'log.syslog.format' => '{%app%} %message%', 
 
 
 Conditional Logging Level Increase
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can configure the logging level to automatically increase to ``debug`` when one of three conditions are met:
+You can configure the logging level to automatically increase to ``debug`` when the first condition inside a condition block is met.
+You can populate one, two or all three conditions.
 
-# ``shared_secret``: If a request parameter with the name ``log_secret`` is set to this value the condition is met.
+ - ``shared_secret``: If a request parameter with the name ``log_secret`` is set to
+               the value of this ``shared_secret``, the condition is met.
+ - ``users``:  If the current request is done by one of the specified users,
+               this condition is met.
+ - ``apps``:   If the log message is invoked by one of the specified apps,
+               this condition is met.
+ - ``logfile``: The log message invoked gets redirected to this logfile 
+	   when a condition above is met.
+	   
+Notes regarding the logfile key:
 
-# ``users``: If the current request is done by one of the specified users, this condition is met.
+1. If no logfile is defined, the standard logfile is used.
+2. Not applicable when using syslog.
 
-# ``apps``: If the log message is invoked by one of the specified apps, this condition is met.
 
-The following example demonstrates what all three conditions look like::
+| The following example demonstrates how all three conditions can look like.
+| The first one that matches triggers the condition block writing the log entry to the defined logfile.
+
+::
 
  'log.condition' => [
 	'shared_secret' => '57b58edb6637fe3059b3595cf9c41b9',
-	'users' => ['sample-user'],
-	'apps' => ['files'],
+	'users' => ['user2'],
+	'apps' => ['gallery'],
+	'logfile' => '/tmp/test2.log'
  ],
 
+| Based on the conditional log settings above:
+| This shows how all requests matching the ``log_secret`` are logged to the logfile.
```

Comment:
s/requests matching the/requests matching/g

Reply:
got it, ok